src/main/java/main/Configurations.java

```diff
@@ -0,0 +1,34 @@
+package main;
+
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import java.util.Properties;
+
+public class Configurations {
+    private static int defaultPosition;
+    private static String pathToOpenFile;
+    private static String pathToSaveFile;
+    private static String dateFormat;
+
+    public static int getDefaultPosition() {
+        return defaultPosition;
+    }
+    public static String getPathToOpenFile() {
+        return pathToOpenFile;
+    }
+    public static String getPathToSaveFile() {
+        return pathToSaveFile;
+    }
+    public static String getDateFormat() {
+        return dateFormat;
+    }
+
+    public Configurations() throws Exception{
+        Properties props = new Properties();
+        props.load(new InputStreamReader(new FileInputStream("/Users/vladislavzhernovii/IdeaProjects/task1/src/recources/config.ini")));
+        defaultPosition = Integer.valueOf(props.getProperty("defaultPosition","0"));
```

Comment:
I can't see where you are using this parameter

Reply:
that's cause of you told me to create this parameter, but i haven't realized that